### PR TITLE
Allow saving of a plain svg file

### PR DESCRIPTION
**Pull request by jpirnay**

### CODE REVIEW

File: meerk40t/core/elements.py [hunk 1]
```diff
@@ -223,30 +223,30 @@ def plugin(kernel, lifecycle=None):
                 "page": "Classification",
                 "section": "_90_Auto-Generation",
             },
-            {
```
Comment (tatarize) [inline meerk40t/core/elements.py:226]: This rather large commenting out doesn't seem to be related or reflected in the PR title and seems like it might want to be a different PR.

File: meerk40t/core/svg_io.py [hunk 2]
```diff
@@ -154,19 +154,22 @@ class SVGWriter:
     @staticmethod
     def save_types():
         yield "Scalable Vector Graphics", "svg", "image/svg+xml", "default"
+        yield "SVG-Plain (no extensions)", "svg", "image/svg+xml", "plain"
```
Comment (tatarize) [inline meerk40t/core/svg_io.py:157]: I basically forgot I spent a while writing this whole save_types protocol.

**Review by tatarize — APPROVED** [review]

File: meerk40t/core/elements.py (hunk 1 above)
Comment (jpirnay) [reply meerk40t/core/elements.py:226]: Agree
